### PR TITLE
ci: Add cypress with example

### PR DESCRIPTION
**Pull request by kykrom90**
## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [x] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes.

## Description

Please describe your changes

## Testing

Please outline all testing steps

1. Pull branch locally and run `yarn` to install new deps
2. etc...

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (technojak) [inline cypress.json:1]: is this file needed since there is a `cypress.json` in the `cypress` directory?

**Review by technojak — APPROVED** [review]
lgtm. just a question about the root level cypress file

File: cypress.json (hunk 1 above)
Comment (kykrom90) [reply cypress.json:1]: This file is needed to execute the framework... In the cypress directory there is another one to be modified in future only if we add a continuous delivery app like CircleCI, Jenkins...

**Review by bootsboogie — APPROVED** [review]
